procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -137,9 +129,9 @@ Make sure the bug that tracks the migration has no blocking items.
 Land whatsnewpage list of locales
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**TODO** - this needs to change, as the process no longer assumes this,
```

Comment:
@jcristau do you think we should remove this section altogether or keep it for now until we have a few successful releases ironing out the new WNP process?

Reply:
I think we're probably OK removing it from the regular merge duty doc given https://bugzilla.mozilla.org/show_bug.cgi?id=1680091#c0